models/buildCluster.js

```diff
@@ -0,0 +1,124 @@
+'use strict';
+
+const Joi = require('joi');
+const mutate = require('../lib/mutate');
+const Command = require('../config/command');
+const scmContext = Joi.reach(require('./pipeline').base, 'scmContext');
+const scmOrgization = Joi
+    .string().max(100)
+    .description('SCM organization name')
+    .example('screwdriver-cd');
+
+const MODEL = {
+    id: Joi
+        .number().integer().positive()
+        .description('Identifier of this Job')
+        .example(123345),
+
+    name: Joi
+        .string().regex(/^[\w-]+$/)
+        .max(50)
+        .description('Name of the build cluster')
+        .example('iOS'),
+
+    description: Joi
+        .string().max(100)
+        .description('Description of the build cluster')
+        .example('Build cluster for iOS team'),
+
+    scmContext,
+
+    scmOrganizations: Joi.array().items(scmOrgization),
+
+    isActive: Joi
+        .boolean()
+        .description('Flag if the the build cluster is active')
+        .example(true)
+        .default(true),
+
+    managedBy: Joi
```

Comment:
`maintainer` for consistency? not sure if that's too vague or not

Reply:
this is not the specific maintainer, it's more like the maintainer type? can either be screwdriver or external

Reply:
ah I see
interesting

Reply:
I agree it's better if it's consistent. @parthasl is working on the naming :D lol

Reply:
if it's only two possibilities, a boolean like `default` might make more sense for this